### PR TITLE
Adding Hive to Runtime spec

### PR DESCRIPTION
**Pull request by alanfgates**
Here's an initial PR for people to review.  This adds Hive to the runtime spec.

### CODE REVIEW

**Comment by rajddesai** [comment]
+1

File: ODPi-Runtime.md [hunk 1]
```diff
@@ -1,18 +1,26 @@
 ODPi Technical Working Group
 
-ODPi Runtime Specification: 1.0
+ODPi Runtime Specification: 1.1
```
Comment (jmertic) [inline ODPi-Runtime.md]: This should be 2.0.

File: ODPi-Runtime.md [hunk 2]
```diff
@@ -234,6 +262,8 @@ custom-to-the-application configuration file, etc) that does not impact the ODPi
 
 -   Applications SHOULD either launch via the Apache Hadoop YARN ResourceManager REST API or via `${HADOOP_YARN_HOME}/bin/yarn jar`
 
+-   Applications SHOULD use JDBC, ODBC, or SQL to determine the structure of metadata rather than directly querying the metastore thrift interface whenever possible.
```
Comment (jmertic) [inline ODPi-Runtime.md]: Is it understood enough that this would be via Hive, or should that be called out?

File: ODPi-Runtime.md [hunk 3]
```diff
@@ -43,19 +54,19 @@ Each entry will have a designation (free text in square brackets) in order to pi
 
 The designation [TEST_ENVIRONMENT] is reserved for entries that are defining constraints on the environment in which ODPi-compliant Hadoop platforms are expected to run. The output of the reference implementation validation testsuite is expected to capture enough information to identify whether the test execution environment conforms to the specification.
 
-Hadoop Build Specifications
+Build Specifications
 ===========================
 
-To help achieve TONE, ODPi-compliant Hadoop platforms MUST conform to the following build specifications.
+To help achieve TONE, ODPi-compliant platforms MUST conform to the following build specifications.
```
Comment (jmertic) [inline ODPi-Runtime.md:60]: Proper terminology should be 'ODPi Runtime Compliant Platforms'

**Review by jmertic — CHANGES_REQUESTED** [review]
Small cleanups needed.

File: ODPi-Runtime.md (hunk 3 above)
Comment (alanfgates) [reply ODPi-Runtime.md:60]: ODPI-compliant is used all over this section of the spec.  If we want to change that we should have a separate issue to change them all.

**Comment by alanfgates** [comment]
Changed version number and made it clear we're talking about Hive metadata, per John's feedback.  Rejected last comment on ODPi-compliant as that is used extensively, not just in that one place.

**Review by jmertic — APPROVED** [review]

**Comment by rvs** [comment]
LGTM. Lets merge!

**Pull request merged**